notmyidea-cms/templates/index.html

```diff
@@ -6,20 +6,20 @@
     {% if loop.index == 1 %}
      <section id="content" class="body">
         <aside id="featured"><article>
-                <h1 class="entry-title"><a href="{{ SITEURL }}/{{ article.url
-                }}">{{ article.title }}</a></h1>
+                <h2 class="entry-title"><a href="{{ SITEURL }}/{{ article.url
+                }}">{{ article.title }}</a></h2>
                 {% include 'article_infos.html' %}<!-- /.post-info -->
                 {{ article.content }}
         </article></aside><!-- /#featured -->
         {% if loop.length  > 1 %}
-                <h1>Other articles</h1>
+	<!--                <h2>Other articles</h2> -->
```

Comment:
I don't think committing commented-out code to the repository is optimal practice. Wouldn't you agree?

Reply:
Umm, no it shouldn't.  Why did I do this???  Ugh, it's been too long.